scripts/ci/bazel-tests.json

```diff
@@ -4,6 +4,12 @@
     {
         "configurations": [
             {"node": "linux-x86_64"},
+            {
+             "node": "linux-x86_64",
+             "parameters": {
+                 "build_opts": ["--python_path=/usr/bin/python3"]
+              }
+            },
```

Comment:
I recommend removing the non-linux configuration so you get a less noisy result. E.g. freebsd is a breakage at HEAD.

Reply:
Done